src/TestInfrastructure/Helpers/MefTestHelpers.cs

```diff
@@ -54,6 +54,44 @@ public static Export CreateExport<T>(object exportInstance, string contractName
             return export;
         }
 
+        /// <summary>
+        /// Overload that imports using a SingleObjectImporter and checks that the expected
+        /// implementing type was imported, and that the expected CreationPolicy has been applied
+        /// </summary>
+        public static SingleObjectImporter<TImportType> CheckTypeCanBeImported<TTypeToCheck, TImportType>(
+            CreationPolicy expectedCreationPolicy,
```

Comment:
Added `expectedCreationPolicy` as a parameter so we can check whether the type is a singleton or not.

The way this method is written means none of the existing exports are being checked. We could assume that most of them are singletons, and tweak these test helpers so that if we don't specify a creation policy we check for a singleton. However, that would probably caused some tests failures that would required fixing up, so I'll do that separately.